azure-pipelines.yml

```diff
@@ -107,25 +107,16 @@ jobs:
       packDirectory: $(Build.ArtifactStagingDirectory)/deployables
 
   - powershell: >-
-      $testdir="bin\Nerdbank.Streams.Tests\$(BuildConfiguration)\net461"
-
-      & "$HOME\.nuget\packages\OpenCover\4.6.519\tools\OpenCover.Console.exe"
-      -register:user
-      -target:"$HOME\.nuget\packages\xunit.runner.console\2.3.1\tools\net452\xunit.console.x86.exe"
-      -targetargs:"$testdir\Nerdbank.Streams.Tests.dll -noshadow -parallel all -notrait SkipInCodeCoverage=true -html $testdir\testresults.html -xml $testdir\testresults.xml -appveyor -nologo"
-      -returntargetcode
-      -excludebyattribute:*.ExcludeFromCodeCoverage*
-      -excludebyfile:*\*Designer.cs
-      -filter:"+[Nerdbank.Streams]*"
-      -hideskipped:All
-      -output:$testdir\code_coverage.xml
+      $testdir="src\Nerdbank.Streams.Tests"
+
+      & dotnet test -c Release --no-build $testdir -f netcoreapp2.2 --filter "TestCategory!=FailsInCloudTest" /p:CollectCoverage=true
```

Comment:
We've already tested both target runtimes in an earlier step. This redundant one existed for legacy reasons. I wonder if we need it at all any more. Can we collect coverage with the `dotnet test` step we're doing earlier in the pipeline? That one runs tests against multiple runtimes, so how does collecting coverage work in such a case?

Reply:
Ideally it generates cc report for each target framework.  However, it makes multi-targeting test project slow, besides that, coverlet seems to run into dll hell together with xunit when targeting netfx instead of netcore (I remember NUnit works with coverlet on netfx).   So I perfer not to fix this comment in this PR but leave it to you to decide how to optimize the ci pipeline, does that make sense. .

Reply:
Sure. Thanks.